Fix: Consistently return object with error property

mediavrog opened the pull request:
Just a small fix to always return an object from the results.

Note: PR should be squash-merged. Sorry didn't rebase

mediavrog committed:
fix: return results as objects consistent

add contributors section

janvogt commented:
Hi @mediavrog, thanks again for the pull request. LGTM and I'm happy to merge it. Could you just add a Contributors section to the README.md and add yourself? If you want to do the rebase still you could also just send a new rebased pull request - whatever you prefer. I think the project is small enough that a few more commits won't be a problem, also my commit messages weren't the best until now ;)

mediavrog commented:
@janvogt Hey, I have rebased and added a contributors section to the readme according to https://github.com/kentcdodds/all-contributors. Enjoy having your face there as well ^^ Cheers

janvogt commented:
Hi @mediavrog, thanks for your contribution. And this all-contributors thing looks really nice. I have to check it out in more detail soon...
